certified-connectors/Datablend/apiDefinition.swagger.json

```diff
@@ -0,0 +1,958 @@
+    "description": "Connect to the Datablend API for a cost-effective iPaaS solution supporting finance and accounting ",
+    "version": "1.0",
+    "contact": {
+      "name": "Datablend Support",
+      "url": "https://datablend.com/contact",
+      "email": "support@datablend.com"
+    }
+  },
+  "host": "api.ml.datablend.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/groups/search": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "success",
+            "schema": {
+              "$ref": "#/definitions/GroupSearch"
+            }
+          }
+        },
+        "summary": "Groups Search",
+        "description": "Groups Search",
+        "operationId": "GroupsSearch",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "offset": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "offset",
+                  "default": 0
+                },
+                "limit": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "limit",
+                  "default": "1"
+                },
+                "orders": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "path": {
+                        "type": "string",
+                        "description": "path",
+                        "default": "name"
+                      },
+                      "direction": {
+                        "type": "string",
+                        "description": "sort direction",
+                        "default": "Ascending"
+                      }
+                    }
+                  },
+                  "description": "orders"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/query-executions/{id}/results": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "success",
+            "schema": {
+              "$ref": "#/definitions/QueryExecutionResults"
+            }
+          }
+        },
+        "summary": "Get Query Execution By Id",
+        "description": "Get Query Execution By Id",
+        "operationId": "GetQueryExecutionById",
+        "x-ms-visibility": "important",
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "The ID of the query execution used when running a workflow.",
+            "description": "Get Query Execution By Id"
```

Comment:
@bmp02050 if you use abbreviations, please make sure to capitalize them to make it clear. Please refer to [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings). The summary above has capitalized ID.

Reply:
Done